### ISSUE TITLE
dotfiles don't work

### ISSUE THREAD

**Issue by sosdos** [1amSimp1e/dots#96]
so after reading and preforming the instructions for the late night dots, and rebooting, hyprland remains the same as it was before, but with a new background.  there were a few errors at the top of the screen, which were related to the style of the config file. I deleted the big titles for the sections, leaving me with the following errors:

Config error in file /home/admin/.config/hypr/hyprland.conf at line 65: cannot parse "6.8" as an int. Config error in file /home/admin/.config/hypr/hyprland.conf
at line 134: Invalid config line 

Config error in file /home/admin/.config/hypr/hyprland.conf at line 142: source globbing error: found no match

Did I do something wrong? Additionally, I don't think these two errors (if fixed) would actually fix the fact that nothing else is present. I know I installed all the packages correctly, them copied all the config files. What else is wrong here?

**Comment by meduk0** [comment]
yeah it a problem just launch a terminal and chnage the config also he miss to add the keybindings config i will try to make my own maybe i will share it with you
view this https://github.com/1amSimp1e/dots/issues/92

**Comment by 1amSimp1e** [comment]
did you fix it ???

**Comment by sosdos** [comment]
yeah sorry I don't check these much, I had  to play around with the keybindings files a bit for it to work, the naming conventions were likely confused at some point during the making of the files

**Comment by JakobLiTe** [comment]
I get these errors, and yes i have nessecary files installed.
Config error in file /home/myuser/.config/hypr/hyprland.conf at line 141: Invalid rulev2 syntax: Rofi
Config error in file /home/myuser/.config/hypr/hyprland.conf at line 141: Invalid rulev2 syntax: pavucontrol
Config error in file /home/myuser/.config/hypr/hyprland.conf at line 141: Invalid rulev2 syntax: Rofi
Config error in file /home/myuser/.config/hypr/hyprland.conf at line 141: Invalid rulev2 syntax: Thunar
Config error in file /home/myuser/.config/hypr/hyprland.conf at line 141: Invalid rulev2 syntax: discord
(4 more...)

**Comment by 1amSimp1e** [comment]
Sorry for late response, if you have that error, that's due to the fact that syntax has been deprecated with the newest update of hyprland. From [this](https://www.reddit.com/r/hyprland/comments/1jjizmk/error_invalid_rulev2_syntax/) reddit post, I would fix it by replacing each line of the error:

`windowrule = tile,^(Rofi)$` to `windowrule = tile,title:(^(Rofi)$)`

**Comment by JakobLiTe** [comment]
I fixed it by changing it to windowrulev2=tile,class:^(rofi)$ after revisiting it! thanks
